lib/rules/no-ternary.js

```diff
@@ -11,6 +11,8 @@
 
 module.exports = {
     meta: {
+        type: "style",
```

Comment:
Since fixing errors reported by this rule would require refactoring the code (albeit a tiny one), should this be a "suggestion"?

Reply:
Agree